app/src/main/java/com/example/root/smsreader/MainActivity.java

```diff
@@ -22,117 +15,116 @@
 
 public class MainActivity extends AppCompatActivity {
 
-public  int len_all_sms = 0;
-public int Count=0;
+    private static final String TAG = MainActivity.class.getSimpleName();
```

Comment:
When you log anything, use TAGs to allows categorization and sorting and filtering later on.

eg- 
`Log.i("MainActivity","Some Msg");` will become `Log.i(TAG,"Some Msg");`

In each Activity, define the TAG string variable using `private static final String TAG = MainActivity.class.getSimpleName();`